### DISCUSSION TITLE
emit open lineage events to a kafka topic

### DISCUSSION THREAD

**Discussion by jayanitw** [OpenLineage/OpenLineage#641] [Q&A]
Hi everyone,
I'm trying to send the Lineage to Azure Purview from spark applications. Can someone help me in setting the kafka config for open lineage for sending events to a atlas hook? 

Thanks,
Jaya.

**Comment by cerveada** [comment]
Not sure if it's what you want, but the [rest-endpoint](https://github.com/AbsaOSS/spline-open-lineage/tree/master/rest-endpoint) from [spline-open-lineage integration](https://github.com/AbsaOSS/spline-open-lineage) is a small web app that just provide open lineage rest API and send all messages arriving there into Kafka topic.

**Reply by collado-mike** [reply]
There's a lightweight version of (I think) the same thing in the proxy application https://github.com/OpenLineage/OpenLineage/tree/main/proxy
